models/_sector/cex/deposit_addresses/chains/cex_arbitrum_deposit_addresses.sql

```diff
@@ -0,0 +1,20 @@
+{% set blockchain = 'arbitrum' %}
+
+{{ config(
+        
+        schema = 'cex_' + blockchain,
+        alias = 'deposit_addresses',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['address']
+)
+}}
+
+{{cex_deposit_addresses(
+        blockchain = blockchain
+        , token_transfers = ref('tokens_' + blockchain + '_transfers')
+        , cex_addresses = ref('cex_' + blockchain + '_addresses')
+        , cex_flows = ref('cex_' + blockchain + '_flows')
+        , first_funded_by = source('addresses_events_' + blockchain + '_first_funded_by')
```

Comment:
sources should be two strings, separated by comma, not the same format as ref one string
i.e. `source('addresses_events_<blockchain>', 'first_funded_by')`

Reply:
ah ok thanks!